extensions/eda/plugins/event_source/pg_listener.py

```diff
@@ -81,6 +24,53 @@
     MESSAGE_XX_HASH,
 )
 
+DOCUMENTATION = r"""
+---
+author:
+  - Doston Toirov (@dtoirov)
+short_description: Read events from pg_pub_sub.
+description:
+  - An ansible-rulebook event source plugin for reading events from pg_pub_sub.
+version_added: '2.4.0'
+options:
```

Comment:
This is wrong. 
Look at the original: https://github.com/ansible/event-driven-ansible/pull/385/files#diff-9044cb1d0c37a42fa4acb893834fdd6be6edc60ce1ded948929e6f78806413abL12

Reply:
corrected.